### PR TITLE
Sis0k0/update speakers sessions

### PR DESCRIPTION
**Pull request by sis0k0**
This PR adds:

- Ilia Idakiev to the speaker's list;
- social links to the Stanimira Vlaeva's info;
- Ilia Idakiev's session: `Deep dive into zone.js`;
- Stanimira Vlaeva's workshop: `From web to mobile with NativeScript and Angular`

### CODE REVIEW

File: docs/default-firebase-data.json [hunk 1]
```diff
@@ -463,6 +479,30 @@
...
+      "name": "Ilia Idakiev",
+      "photo": "gs://dev-fest-sofia-2019.appspot.com/speakers/IliaIdakiev.jpeg",
+      "photoUrl": "https://firebasestorage.googleapis.com/v0/b/dev-fest-sofia-2019.appspot.com/o/speakers%2FIliaIdakiev.jpeg?alt=media&token=6919e2c8-807a-4235-afe8-4377ff004e9f",
+      "shortBio": "Google Developer Expert (GDE), software developer-entrepreneur, as well as a JavaScript enthusiast who has many years of experience in web application development.",
+      "socials": [{
+        "icon": "twitter",
+        "link": "https://twitter.com/ilia_idakiev",
+        "name": "Twitter"
+      }],
+      "order": 7,
```
Comment (bstrandjev) [inline docs/default-firebase-data.json]: Please, as is traditional for such events, make the session lecturers of higher order than workshop leaders. Also, please remove the featured flag for all non-main track lecturers.

Comment (sis0k0) [reply docs/default-firebase-data.json]: Which one is the main track - the sessions track or the track with `Lectures 1, 4, 5, and 7`?

**Comment by bstrandjev** [comment]
Both lecture tracks are main, or at least this is what i meant. Only the
workshops are not

File: docs/default-firebase-data.json (hunk 1 above)
Comment (bstrandjev) [reply docs/default-firebase-data.json]: Both lecture tracks are main, or at least this is what i meant. Only the workshops are not

**Review by bstrandjev — APPROVED** [review]
I just realized i have asked to put your own self as non-featured speaker. How big of an event should ours be to afford not to feature a GDE :) This is definitely not a strive to understate the workshop leader's participation in the event, rather the main track speakers will reach more of an audience

**Comment by sis0k0** [comment]
That's totally ok with me. We've got an amazing speaker line-up!

**Pull request merged**